eslint/index.js

```diff
@@ -1,16 +1,13 @@
 module.exports = {
 	env: {
 		browser: true,
-		es6: true
+		es6: true,
 	},
-	extends: ['eslint:recommended', 'plugin:import/recommended', 'prettier'],
-	plugins: ['html'],
-	parser: '@babel/eslint-parser', // so dynamic `import` is recognized
```

Comment:
The default eslint parser is pretty good so i revert to it for JS. Tested it in FE and its ok.